tests/test_parse.py

```diff
@@ -1378,6 +1378,24 @@ def test_parsable_annotation_order(input):
     assert block.annotated_ast_node
 
 
+@pytest.mark.skipif(sys.version_info < (3, 6), reason="invalid early python syntax")
+@pytest.mark.parametrize(
+    "input",
+    [
+        '''x='abc'
+f"""\
+This is a
+multi-line
+f-string
+with input {x}.\
```

Comment:
What about a test for when the string has a more complicated expression. It should at least not crash pyflyby.